File: src/main/java/com/sothawo/springdataelastictest/products/ProductRepository.java
/*
 * (c) Copyright 2021 sothawo
 */
package com.sothawo.springdataelastictest.products;

import org.springframework.data.elasticsearch.repository.ElasticsearchRepository;

/**
 * @author P.J. Meisch (pj.meisch@sothawo.com)
 */
public interface ProductRepository extends ElasticsearchRepository<Product, String> {
}

File: src/main/kotlin/com/sothawo/springdataelastictest/products/ProductRepository.kt
package com.sothawo.springdataelastictest.products

import org.springframework.data.elasticsearch.repository.ElasticsearchRepository

/**
 * @author P.J. Meisch (pj.meisch@sothawo.com)
 */
interface ProductRepository : ElasticsearchRepository<Product, String>
